features/AssetController.feature

```diff
@@ -26,6 +26,15 @@ Feature: DeviceManager asset controller
       | model           | "PERFO"         |
       | metadata.foobar | 42              |
       | metadata.index  | "engine-kuzzle" |
+    When I successfully execute the action "device-manager/asset":"search" with args:
```

Comment:
You should takea look at the Kuzzle Core functional tests that use `document:search`: https://github.com/kuzzleio/kuzzle/blob/b58fa9b4b133ca92811fc6d7102fa5672cd35400/features/DocumentController.feature#L45